### ISSUE TITLE
Posts em destaque

### ISSUE THREAD

**Issue by eduardomarqu-es** [MacMagazine/app-iOS#184]
Assim como no site existem seis posts destacados (uma fileira com três logo no começo e outra fileira com três logo abaixo), podemos deixar a área superior para destacar esses artigos.

**Comment by rfischmann** [comment]
É, acho que já tivemos isso numa versão antiga do app. Não lembro por que tiramos, mas faz sentido. Normalmente, em mobile, o jeito é fazer um "slider" mesmo.

**Comment by kazziosoftware** [comment]
Normalmente se faz um horizontal scroll no topo. A pergunta é: mostrar todos? Mostrar quantos?
Faz sentido mostrar um scroll horizontal ou ter uma sessão específica (mais um botão na base)?
Juntar com #183, não seria mais lógico? Já que haverá filtro por categoria, uma das categorias seria "Somente Destaques"?

**Comment by rfischmann** [comment]
Sim, o "slider" que falei era o "horizontal scroll" no topo (com aquelas bolinhas logo abaixo indicando quantos têm). Eu mostraria 6, tal como temos no site em si.

Não gosto da ideia de juntar com o #183, acho que são coisas diferentes.

**Comment by maclacerda** [comment]
Boa tarde pessoal, já temos algum dev trabalhando nessa demanda?
Caso negativo, gostaria de pega-la para desenvolver, posso propor um "conceito" e enviar um video / imagem por aqui para vocês darem uma olhada e se for alinhado com o desejo de vocês seguimos em frente ou podemos discutir uma melhor alternativa.

Abraços

**Comment by rfischmann** [comment]
Oi, @maclacerda! Não há ninguém trabalhando ativamente nisso, não. Pode mandar ver nesse conceito, meu caro! 😊 Valeu. Abraços.

**Comment by cassio-rossi** [comment]
@maclacerda se vc tiver Telegram, me add (@cassiorossi) assim podemos trocar mensagens e ideias.

**Comment by maclacerda** [comment]
Te chamo aí jaja @cassio-rossi 
Tenho algumas sugestões daí agente discute 🙃
